packages/zendesk-adapter/src/filters/help_center_section.ts

```diff
@@ -0,0 +1,100 @@
+/*
+*                      Copyright 2022 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import {
+  Change,
+  Element,
+  getChangeData,
+  InstanceElement,
+  isInstanceElement,
+} from '@salto-io/adapter-api'
+import _ from 'lodash'
+import Joi from 'joi'
+import { createSchemeGuard } from '@salto-io/adapter-utils'
+import { FilterCreator } from '../filter'
+import { deployChange, deployChanges } from '../deployment'
+
+const SECTION_TYPE_NAME = 'section'
+
+type TranslationType = InstanceElement &{
+    title: string
+    body?: string
+    locale: string
+}
+
+export type sectionType = InstanceElement & {
+  locale: string
+}
+
+const TRANSLATION_SCHEMA = Joi.object({
+  locale: Joi.string().required(),
+  body: Joi.string(),
+  title: Joi.string().required(),
+}).unknown(true).required()
+
+export const isTranslation = createSchemeGuard<TranslationType>( // TODO
+  TRANSLATION_SCHEMA, 'Received an invalid value for translation'
+)
+
+const addTranslationValues = (change: Change<InstanceElement>): void => {
+  const currentLocale = getChangeData(change).value.source_locale
+  const translation = getChangeData(change).value.translations
+    .filter(isTranslation)
+    .find((tran:TranslationType) => tran.locale === currentLocale)
```

Comment:
```suggestion
    .find((tran: TranslationType) => tran.locale === currentLocale)
```
also - I think we will want to convert the locales to references eventually and then this will need to change (but we can change that later as well)